redash/handlers/query_results.py

```diff
@@ -64,7 +64,9 @@ def run_query_sync(data_source, parameter_values, query_text, max_age=0):
         return None
 
 
-def run_query(data_source, parameter_values, query_text, query_id, max_age=0):
+def run_query(data_source, parameter_values, query_text, query_id, max_age=0, parameter_schema=None):
+    if parameter_schema is None:
+        parameter_schema = {}
```

Comment:
```suggestion
        parameter_schema = []
```

No?

Reply:
you're right, but I think I have something better